app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -37,5 +48,6 @@ export default class extends Controller {
       else
         console.error(`unable to find target for ${key}`)
     }
+    window.scrollTo(0, 80)
```

Comment:
@jcoyne what's this doing?

Reply:
Scrolls back to the top when errors are present.